components/css/buckyless/md-generated.less

```diff
@@ -260,3 +230,8 @@ md-tooltip {
     }
   }
 }
+
+// Adjust z-index of tooltips
+.md-panel-outer-wrapper.md-panel-is-showing {
+  z-index: 54 !important;
+}
```

Comment:
[The linter flags this missing end-of-file newline](https://travis-ci.org/github/uPortal-Project/uportal-app-framework/builds/679020979#L759). [editorconfig](https://editorconfig.org/) should be [automating this bit of style](https://github.com/uPortal-Project/uportal-app-framework/blob/master/.editorconfig#L28). Opportunity to configure local tools to honor editorconfig?

Reply:
I made the change in .editconfig!